addons/web/static/src/scss/bootstrap_review.scss

```diff
@@ -80,6 +80,21 @@
     }
 }
 
+// Do not display the backdrop element added by bootstrap in the body and added a
+// background on the modal to keep the same effect. The bootstrap backdrop was useful
+// for compatibility with <IE9 but is no longer needed. This also avoids z-index
+// issues because the wrapwrap (z-index: 0) can contain a modal. And in this case,
+// the backdrop (z-index: 999) goes above the modal.
+.modal-backdrop {
+    display: none;
+}
+.modal:not(.o_modal_no_backdrop) {
+    background-color: rgba($modal-backdrop-bg, $modal-backdrop-opacity);
+}
+.modal[data-backdrop="false"] {
```

Comment:
If you can check the `[data-backdrop]` selector, why do you need the o_modal_no_backdrop class?